processes/weekly-content-sharing.md

```diff
@@ -0,0 +1,24 @@
+# Weekly Content Sharing
+We're planning to showcase Carvel by publishing new content weekly. This will
+increase the project's visibility so that we can grow the community.
+
+We'd like this to be a lightweight effort (i.e. less formal that documentation).
+The topic being covered can be anything Carvel-related. Find something that's
+interesting to you. Some ideas include use case walkthroughs (one tool,
+multi-tool, w/ non-Carvel tools), concept explanations, introducing new
+features, or highlighting existing features. To broadcast awareness of our
+publishings, we'll tweet and share them at community meetings.
+
+Below is a signup for individuals to reserve weeks in advance.
```

Comment:
@microwavables `carvel-dev@googlegroups.com` doesn't feel like the right place to send contact information since that's a public distribution list. What about DMing someone on Slack or a more restricted distribution list?

Reply:
good point - I updated to have them email me directly